backend/EasySpeak.Core/EasySpeak.Core.WebAPI/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -25,19 +25,25 @@ public static void RegisterCustomServices(this IServiceCollection services, ICon
         {
             services
                 .AddControllers()
-                .AddNewtonsoftJson(options => options.SerializerSettings.ReferenceLoopHandling = Newtonsoft.Json.ReferenceLoopHandling.Ignore);
+                .AddNewtonsoftJson(options =>
+                    options.SerializerSettings.ReferenceLoopHandling = Newtonsoft.Json.ReferenceLoopHandling.Ignore);
 
             services.AddTransient<ILessonsService, LessonsService>();
             services.AddTransient<IZoomApiService, ZoomApiService>();
             services.AddScoped<IFriendService, FriendService>();
-            services.AddSingleton<IConnectionProvider>(_ => new ConnectionProvider(configuration.GetValue<string>("Rabbit")));
+            services.AddSingleton<IConnectionProvider>(_ =>
+                new ConnectionProvider(configuration.GetValue<string>("Rabbit")));
             services.AddTransient<IMessageProducer, MessageProducer>();
             services.AddTransient<IHttpRequestService, HttpRequestService>();
             services.AddScoped<IFirebaseAuthService, FirebaseAuthService>();
-            services.AddScoped<IUserService, UserService>();
             services.AddScoped<IChatService, ChatService>();
             services.AddFirebaseApp();
-            services.AddScoped<INotificationService, NotificationService>();
+            var test = new RabbitQueuesOptions();
+            configuration.GetSection("RabbitQueues").Bind(test);
```

Comment:
Do you need this?

Reply:
Yeah, If I'm not wrong, I can't register GraphDatabase without passing info from this section, I've tried to do this, but anyway, I'll check it out

Reply:
No, I definitely don't need this)